examples/ra-sim/main.py

```diff
@@ -0,0 +1,87 @@
+from argparse import ArgumentParser
+from time import perf_counter
+from typing import Any, Dict, Tuple
+
+import gym
+gym.logger.set_level(40)
```

Comment:
Co Robi ta funkcja `set_level()`?

Reply:
Sprawia, że gym nie pluje tysiącem warningów o tym, że na przykład zwracam liczbę albo `jnp.array` zamiast tablicy w numpy ;) Gym jakoś dziwnie sprawdza kompatybilność zwracanych typów i uznaje, że gdy zwracam liczbę w miejscu `Box` o kształcie (1,) to jest źle, bo powinienem zwracać tablicę w numpy.

Reply:
Dzięki!